sensors:

  logical_camera_conveyor:
    type: logical_camera
    pose:
      xyz: [-0.53, 1, 1.820000]
      rpy: [0, 1.570796, 'pi']
  logical_camera_1_2:
    type: logical_camera
    pose:
      xyz: [-2.3, 3.0, 1.820000]
      rpy: [0, 1.570796, 'pi']
  logical_camera_3_4:
    type: logical_camera
    pose:
      xyz: [-2.3, -3.0, 1.820000]
      rpy: [0, 1.570796, 'pi']      
  
  logical_camera_station1:
    type: logical_camera
    pose:
      xyz: [-7.262095, 2.963994, 2.285697]
      rpy: [3.141593, 1.570792, 0.000000]

  logical_camera_station2:
    type: logical_camera
    pose:
      xyz: [-12.26217, 2.963994, 2.285697]
      rpy: [3.141593, 1.570792, 0.000000]

 
  logical_camera_station3:
    type: logical_camera
    pose:
      xyz: [-7.262095, -2.963994, 2.285697]
      rpy: [3.141593, 1.570792, 0.000000]

  logical_camera_station4:
    type: logical_camera
    pose:
      xyz: [-12.26217, -2.963994, 2.285697]
      rpy: [3.141593, 1.570792, 0.000000]      

#robot_camera:
# enable: true

  # break_beam_1:
  #   type: break_beam
  #   pose:
  #     xyz: [1.6, 2.25, 0.91]
  #     rpy: [0, 0, 'pi']
  # proximity_sensor_1:
  #   type: proximity_sensor
  #   pose:
  #     xyz: [0.95, 2.6, 0.92]
  #     rpy: [0, 0, 0]
  # logical_camera_1:
  #   type: logical_camera
  #   pose:
  #     xyz: [-0.3, 0.383, 1.27]
  #     rpy: [0.2, 1.5707, 0]
  # logical_camera_2:
  #   type: logical_camera
  #   pose:
  #     xyz: [0.3, 3.15, 1.5]
  #     rpy: [0, 'pi/2', 0]
  # logical_camera_3:
  #   type: logical_camera
  #   pose:
  #     xyz: [0.3, -3.15, 1.5]
  #     rpy: [0, 'pi/2', 0]
  # depth_camera_1:
  #   type: depth_camera
  #   pose:
  #     xyz: [-0.3, -0.383, 1.2]
  #     rpy: [0, 1.5707, 0]
  # laser_profiler_1:
  #   type: laser_profiler
  #   pose:
  #     xyz: [1.21, 4, 1.6]
  #     rpy: ['-pi', 'pi/2', 'pi/2']
